multiscanner/distributed/api.py

```diff
@@ -510,7 +511,7 @@ def create_task():
                 HTTP_BAD_REQUEST)
 
         return make_response(
-            jsonify({'Message': {'task_ids': [task_id]}}),
+            jsonify({'task_ids': [task_id]}),
```

Comment:
Do we need the `task_ids` wrapper here or should we just jsonify the list, like we do for tags and notes?